OpenRA.Mods.Common/UtilityCommands/LegacyMapImporter.cs

```diff
@@ -123,7 +124,11 @@ public static Map Import(string filename, string mod, Ruleset rules, Action<stri
 		{
 			var map = new LegacyMapImporter(filename, rules, errorHandler).map;
 			map.RequiresMod = mod;
-			map.MakeDefaultPlayers();
+
+			var players = new MapPlayers();
```

Comment:
My understanding of this is that you don't import players, but spawn points (at least D2k does it this way and the code there is the same).
After that you use `MakeDefaultPlayers()` to add the PlayerReference definitions to the map in order to be able to use those spawn points.
So yes, this creates an empty collection of players, but there is nothing to be deleted at this point anyway.